lib/extractors/pptx.js

```diff
@@ -38,7 +39,9 @@ function _calculateExtractedText( slideText ) {
         }
       }
     });
-    text += localText + '\n';
+    if (localText !== '') {
```

Comment:
I know textract has a preserve newlines functionality, do you know how forcing a newline here will play if a someone wants `forceNewlines=false`?

Also the coding style of the if doesn't match the other if conditions :)

Reply:
As discussed, the line break before the notes will be left alone and the if condition is removed.